tests/Integration/PrestaShopBundle/Form/EventListener/FeatureValueListenerTest.php

```diff
@@ -313,13 +323,16 @@ class SimpleFeaturesFormTest extends CommonAbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add('feature_value_id', ChoiceType::class, [
-            // these are the default options which should change after listener is called
+            // these are the default options which are important and might be affected by listener
             'disabled' => true,
             'attr' => [
                 'disabled' => true,
                 'data-toggle' => 'select2',
                 'class' => 'feature-value-selector',
             ],
+            // empty data = 0 is important. Without it being set to 0, the "invalid option" constraint is not triggered,
+            // and the form is not invalidated (which should happen because field is not optional therefore 0 is invalid option)
+            'empty_data' => 0,
```

Comment:
this is the important part. It allows invalidating placeholder value.
Without this, the form is considered valid (because the default null value is understood as optiona/placeholder value by symfony). With this, once placeholder is submitted it is converted to 0, which causes invalid option error as expected.